internal/k8s/names.go

```diff
@@ -5,8 +5,14 @@ import (
 	"strings"
 )
 
+const FmtduplicateYamlDetectedError = "Duplicate YAML Entity(s): [%s] has been detected across one or more resources.  Only one specification per entity can be applied to the cluster; to ensure expected behavior, remove the duplicate specifications."
```

Comment:
This is a very silly wording correction but the current implementation will only ever error about one duplicate at a time, so you don't need `entity(s)`, just `entity` is fine
```suggestion
const FmtduplicateYamlDetectedError = "Duplicate YAML Entity: "%s" has been detected across one or more resources.  Only one specification per entity can be applied to the cluster; to ensure expected behavior, remove the duplicate specifications."
```